### ISSUE TITLE
Discuss how similar `null` and non-existence should be in changefeeds.

### ISSUE THREAD

**Issue by mlucy** [rethinkdb/rethinkdb#5215]
Currently we treat them very similarly, but @danielmewes mentioned in a review that he thinks `null` should be treated more like a value and less like non-existence.  At some point we should talk about whether or not we want to change that and what the API would look like if we did.

**Comment by OtterCode** [comment]
I just want to second the idea that null should be treated like a value, given its support and definition in the JSON standard. It would mean that we would have much closer parity between JSON and the way rethinkDB stores data.

**Comment by VeXocide** [comment]
I agree with @danielmewes and @OtterCode, `null` is a JSON value and distinct from the non-existence of a value.
